delocate/delocating.py

```diff
@@ -157,7 +165,7 @@ def _analyze_tree_libs(
             # @rpath, etc, at this point should never happen.
             raise DelocationError("%s was expected to be resolved." % required)
         r_ed_base = basename(required)
-        if relpath(required, rp_root_path).startswith(".."):
+        if Path(rp_root_path) not in Path(required).parents:
```

Comment:
```suggestion
        if not Path(required).is_relative_to(rp_root_path):
```

Reply:
`is_relative_to` is new in 3.9 and it looks like you're testing back to 3.7.

Reply:
Oops. That's easy to miss when I'm not using an IDE.

Python 3.8 EOL is this year. Maybe I can justify dropping support for it? The `path in path.parents` syntax looks terrible even compared to `relpath(...).startswith("..")`.

Reply:
I suppose one could say the syntax isn't great, but the docs do list `root in path.parents` as equivalent to `is_relative_to`.